sphinxcontrib/spelling/builder.py

```diff
@@ -61,15 +61,35 @@ def init(self):
             f.append(filters.ImportableModuleFilter)
         f.extend(self.config.spelling_filters)
 
-        project_words = os.path.join(self.srcdir,
-                                     self.config.spelling_word_list_filename)
-        self.info('Looking for custom word list in {}'.format(
-            project_words))
+        if not os.path.isdir(self.outdir):
+            os.mkdir(self.outdir)
+
+        #
+        # In case the user has multiple word lists, we combine them into one
+        # large list that we pass on to the checker.
+        #
+        word_list = self.config.spelling_word_list_filename
```

Comment:
The old version joined self.srcdir with this value to build the full path. Don't we need to keep that?

Reply:
Oops, that's a typo! Of course we should keep that!

Reply:
Actually, I just realized it was not a typo, because it was joined with srcdir later on, but only in the list case. I'll make sure it works as it used to with only one word list as well.